datasetSizingOps.tex

```diff
@@ -0,0 +1,16 @@
+\tiny \begin{longtable} { |p{0.22\textwidth}  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r |} 
+\caption{Dataset sizes used to calculate storage needs during Operations \label{tab:datasetSizingOps}}\\ 
+\hline 
+\textbf{Dataset Sizing}&\textbf{unit}&\textbf{LOY1/ FY23}&\textbf{LOY2/ FY24}&\textbf{LOY3/ FY25}&\textbf{LOY4/ FY26}&\textbf{LOY5/ FY27}&\textbf{LOY6/ FY28}&\textbf{LOY7/ FY29}&\textbf{LOY8/ FY30}&\textbf{LOY9/ FY31}&\textbf{LOY10/ FY32}&\textbf{Notes} \\ \hline
```

Comment:
I suggest not putting the notes int he table int the doc - rather put them in the text this will make the table fit at least landscape from what i see.  I had left notes out on some other tables also . They remain in the sheet just not in the doc.  This applies to all the ops tables.

Reply:
I removed the notes, adjusting the number of columns in the spreadsheet and the range extracted in the Makefile.